bot/seasons/evergreen/bookmark.py

```diff
@@ -17,17 +20,27 @@ def __init__(self, bot: commands.Bot):
 
     @commands.command(name="bookmark", aliases=("bm", "pin"))
     async def bookmark(
-        self,
-        ctx: commands.Context,
-        target_message: discord.Message,
-        *,
-        title: str = "Bookmark"
+            self,
+            ctx: commands.Context,
+            target: Union[discord.Message, discord.User],
+            *,
+            title: str = "Bookmark"
     ) -> None:
-        """Send the author a link to `target_message` via DMs."""
+        """
+        Send the author a link to `target_message` via DMs.
+
+        A user mention can also be passed to get the user's last message in channel where the command is invoked.
+        """
+        if type(target) == discord.User:
+            async for message in ctx.channel.history(limit=20):  # Takes last 2 messages from the channel.
+                if message.author == target and message != ctx.message:
```

Comment:
Should compare message.id rather than message object.
IDK is that an issue comparing message object not working as it should be.

Reply:
You will have to check the docs if the class supports equality checking, for example [discord.TextChannel](https://discordpy.readthedocs.io/en/latest/api.html#textchannel) supports it
![image](https://user-images.githubusercontent.com/19921765/78340524-737d8d00-75c0-11ea-817d-aae4c39f1678.png)
But [discord.Message](https://discordpy.readthedocs.io/en/latest/api.html#message) does not, so yes you need to check via `.id`